Monitoring in a Box/Ansible/roles/mib_setup/tasks/transfer_mib_file.yaml

```diff
@@ -1,5 +1,5 @@
 - name: Transfer docker files into VM
-  copy:
+  synchronize:
```

Comment:
How much faster is this?

Reply:
very significant. Did not actually time it.